unit3-server-mvc-with-node-express-and-sql/README.md

```diff
@@ -2,50 +2,58 @@
 
 ### Week 1
 
+- Serverside MVC with Express: 3 days
+  - Why MVC for server side development: Separation of concerns
+  - Routing & (simple) regular expressions
+  - Controllers
+  - Views using a templating engine and ViewModels (Mustache? Jade?)
+  - Middleware for session, error handling
+  - **Learning outcome**: Given a small existing website, add new URLs, add content to existing URLs, create a new URL with a new controller that reuses an existing view
+  - **Learning outcome**: Create a website with over 4 unique URL routes with both HTML and JSON responses, user state stored in sessions, and an interactive set of client pages using AJAX to hit at least one JSON endpoint
+- Trees & Graphs: 2 days (Carries over into the next week)
```

Comment:
What do you think with pushing Trees and Graphs to Week 2 and doing Recursion Week 1? Feel like Trees and Graphs might fit in/flow a little better with the database stuff + could potentially use some recursion knowledge when thinking about traversals